students/Prabhat.md

```diff
@@ -0,0 +1,6 @@
+# Prabhat Ojha
+
+## UIET 2nd year
+
+### Did C++ and Data Structures from coding ninjas
```

Comment:
Trying to approve.

Reply:
are there any changes that you want me to make ?